test/test_genpy_generator.py

```diff
@@ -166,8 +166,8 @@ def test_default_value():
         assert '[0,0,0,0]' == default_value(msg_context, t+'[4]', 'std_msgs')
         assert '[0]' == default_value(msg_context, t+'[1]', 'roslib')
 
-    assert "chr(0)*1" == default_value(msg_context, 'uint8[1]', 'roslib')
-    assert "chr(0)*4" == default_value(msg_context, 'uint8[4]', 'roslib')
+    assert b'\0' == eval(default_value(msg_context, 'uint8[1]', 'roslib')
```

Comment:
Thanks! I had no idea why the marker was pointing at the start of the line in the traceback